sources/core/Stride.Core.Reflection/MemberDescriptors/PropertyDescriptor.cs

```diff
@@ -39,17 +39,17 @@ public PropertyDescriptor(ITypeDescriptor typeDescriptor, PropertyInfo propertyI
 
         public override bool HasSet => setMethod != null;
 
-        public override object Get(object thisObject)
+        public override object? Get(object thisObject)
         {
-            return getMethod.Invoke(thisObject, null);
+            return getMethod?.Invoke(thisObject, null);
         }
 
-        public override void Set(object thisObject, object value)
+        public override void Set(object thisObject, object? value)
         {
             if (!HasSet)
                 throw new InvalidOperationException($"The property [{Name}] of type [{DeclaringType.Name}] has no setter.");
 
-            setMethod.Invoke(thisObject, new[] {value});
+            setMethod!.Invoke(thisObject, [value]);
```

Comment:
I think this would be cleaner as 
```cs
if (setMethod is null)
    throw new InvalidOperationException($"The property [{Name}] of type [{DeclaringType.Name}] has no setter.");

setMethod.Invoke(thisObject, [value]);
```
That way, if the code ever changes, which is very unlikely but still, nullability would still be validated for the invoke instead of always ignored

Reply:
now its nullable

Reply:
Well, now you just made it more confusing ... I would rather you use your previous one or my suggestion, sorry :(

Reply:
oh now i get it

Reply:
[854cbfb](https://github.com/stride3d/stride/pull/2266/commits/854cbfb8f7a74fba9d9ae797c3c523946f9ca6db)
made it now to your suggestion